services/offers_service.py

```diff
@@ -1,6 +1,10 @@
 from datetime import date, datetime
 from matplotlib import pyplot as plt, dates as mdates
 from helpers import programmes_helper
+import sys
+sys.path.insert(1,'/path/to/application/app/folder')
+from constant import year
```

Comment:
```suggestion
from constants import year
```
There is a typo here, it should be `constants` instead of `constant`